### PR TITLE
Ipc4:  fix src bugs on windows

### PR DESCRIPTION
**Pull request by RanderWang**

### CODE REVIEW

**Comment by lgirdwood** [comment]
@tmleman @marcinszkudlinski any comments ?

File: src/audio/src/src.c [hunk 1]
```diff
@@ -657,14 +657,13 @@ static int src_params(struct comp_dev *dev,
 
 	comp_info(dev, "src_params()");
 
+	src_set_params(dev, params);
```
Comment (marcinszkudlinski) [inline src/audio/src/src.c:660]: why set before verify?

Comment (RanderWang) [reply src/audio/src/src.c:660]: param is figured out by module config so we need to first set up param then verify param.
For IPC3 path, the param is sent by host driver

Comment (lgirdwood) [reply src/audio/src/src.c:660]: @RanderWang best to state as a comment so that it's easy to follow.

Comment (lgirdwood) [reply src/audio/src/src.c:660]: @RanderWang best to state as a comment so that it's easy to follow.

Comment (RanderWang) [reply src/audio/src/src.c:664]: sure, updated

**Review by tmleman — APPROVED** [review]

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**